purchases/src/main/kotlin/com/revenuecat/purchases/EntitlementInfo.kt

```diff
@@ -0,0 +1,136 @@
+package com.revenuecat.purchases
+
+import android.os.Parcel
+import android.os.Parcelable
+import java.util.Date
+
+/**
+ * @property identifier The entitlement identifier configured in the RevenueCat dashboard.
```

Comment:
Can you add some docs that just gives a quick overview of the purpose of the class and how it relates to the old `Entitlement` class?